pandunia-lekse.csv

```diff
@@ -2772,6 +2822,7 @@ sikle|||eng:bicycle, fra:bicyclette, tur:bisiklet, spa:bicicleta, tgl:bisikleta,
 sikliste||||cyclist|cycliste|ciclista|ciclista|велосипедист||骑自行车的人|チャリダー|||||||bisikletçi|biciklisto|pyöräilyn harrastaja|rowerzysta
 sile|||zho:丝 (sī), yue:絲 (si1), wuu:絲 (sr1), eng:silk, spa:por:sérico, rus:шёлк (sholk)|silk|soie|seda|seda|шёлк||丝绸|絹||||||||||
 silike|mate|Si||silicon||silicio|||||||||||||silicio|pii|krzem
+silke|||ara:سلق (silqa), spa:por:acelga|chard|blettes (bettes)|acelga||мангольд|||フダンソウ (スイスチャード)||||||||||
```

Comment:
I would propose "selge" because Arabic q becomes frequently g and Arabic is really fuzzy with vowels, so Spanish and Portuguese ride over it.

Reply:
Good point!